stripe/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -1097,10 +1097,13 @@ class CardInputWidget @JvmOverloads constructor(
 
     private val peekCardText: String
         get() {
-            return when (brand) {
-                CardBrand.AmericanExpress -> PEEK_TEXT_AMEX
-                CardBrand.DinersClub -> PEEK_TEXT_DINERS_14
-                else -> PEEK_TEXT_COMMON
+            return when (cardNumberEditText.panLength) {
+                19 -> 3
+                15 -> 5
+                14 -> 2
+                else -> 4
+            }.let { peekSize ->
+                List(peekSize) { "0" }.joinToString(separator = "")
```

Comment:
nit: `"0".repeat(peekSize)` is probably simpler. I know you already used this pattern in another PR that I didn't comment on 😬 Didn't really think about it at the time

Reply:
I'll fix in follow up PR